tostadas/0.2.0-beta/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM nfcore/base:2.1 as app
+# nfcore/base is Debian
+
+# comes with warning about conda version
+# ==> WARNING: A newer version of conda exists. <==
+#   current version: 4.9.2
+#   latest version: 22.11.1
+#
+# Please update conda by running
+#
+#     $ conda update -n base -c defaults conda
+
+ARG TOSTADAS_VER="0.2.0-beta"
+
+LABEL authors="Cole and Ankush Gupta"
+LABEL base.image="nfcore/base:2.1"
+LABEL dockerfile.version="1"
+LABEL software="tostadas"
+LABEL software.version=$TOSTADAS_VER
+LABEL description="Image for the TOSTADAS: Toolkit for Open Sequence Triage, Annotation and DAtabase Submission pipeline"
+LABEL website="https://github.com/CDCgov/tostadas"
+LABEL license="https://github.com/CDCgov/tostadas/LICENSE"
+LABEL maintainer="Ankush Gupta"
+LABEL maintainer.email="ankushkgupta@deloitte.com"
+LABEL maintainer2="Kyle O'Connell"
+LABEL maintainer2.email=""
+LABEL maintainer3="Cole Tindall"
+LABEL maintainer3.email=""
```

Comment:
This is missing some email information

Reply:
Just pushed these changes to add the email information for Kyle and Cole